project/frontend/src/components/SwipeMatchLeaderboard.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import FlipMove from 'react-flip-move';
+import RestaurantRankingView from '../components/RestaurantRankingView.js';
+
+function SwipeMatchLeaderboard(props) {
+  const { restaurants } = props;
+
+  return (
+    <FlipMove
+      easing='cubic-bezier(0.35, 0.91, 0.33, 0.97)'
+      staggerDurationBy={15}
+      staggerDelayBy={20}>
+      {[...restaurants]
+        .sort((r1, r2) => r2.fractionLiked - r1.fractionLiked)
+        .map((restaurant, index) => {
+          if (index > 4) return null;
```

Comment:
nit: maybe pull 4 out into a named constant above, - does it represent max # of restaurants shown on the leaderboard ?

Reply:
Done!